src/ui.c

```diff
@@ -27,15 +33,29 @@ SDL_Texture* lcd_tex = NULL;
 SDL_Rect lcd_srcRect = { 0, 0, 320, 240 };
 SDL_Rect lcd_dstRect = { 0, 0, LOGICAL_WIDTH, LOGICAL_HEIGHT };
 
+// LED
+#define UI_LED_IMAGE_SIZE 32
+extern const uint8_t led_png[];
+extern const uint16_t led_png_size;
```

Comment:
Why are these extern if they're defined in the header file?

Reply:
Good point, I had this 2 different ways and then apparently went half-way with both. Fixing.